components/map/map.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import MapView from "react-native-maps";
+import { StyleSheet, Dimensions } from "react-native";
+
+const Map = props => {
+  return <MapView style={styles.mapStyle} region={props.region} />;
+};
+
+getInitialState = async () => {
+  return {
+    region: {
+      latitude: 45.495869,
```

Comment:
Same as before, should be stored in a separate file

Reply:
hmm I don't think this is needed at all